home/jobs/OMERO-web/config.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project>
+  <actions/>
+  <description></description>
+  <keepDependencies>false</keepDependencies>
+  <properties>
+    <jenkins.plugins.slack.SlackNotifier_-SlackJobProperty plugin="slack@1.8.1">
+      <teamDomain></teamDomain>
+      <token></token>
+      <room></room>
+      <startNotification>true</startNotification>
+      <notifySuccess>true</notifySuccess>
+      <notifyAborted>true</notifyAborted>
+      <notifyNotBuilt>true</notifyNotBuilt>
+      <notifyUnstable>true</notifyUnstable>
+      <notifyFailure>true</notifyFailure>
+      <notifyBackToNormal>true</notifyBackToNormal>
+      <notifyRepeatedFailure>false</notifyRepeatedFailure>
+      <includeTestSummary>false</includeTestSummary>
+      <showCommitList>false</showCommitList>
+      <includeCustomMessage>false</includeCustomMessage>
+      <customMessage></customMessage>
+    </jenkins.plugins.slack.SlackNotifier_-SlackJobProperty>
+  </properties>
+  <scm class="hudson.scm.NullSCM"/>
+  <assignedNode>web</assignedNode>
+  <canRoam>false</canRoam>
+  <disabled>false</disabled>
+  <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>
+  <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
+  <triggers/>
+  <concurrentBuild>false</concurrentBuild>
+  <builders>
+    <hudson.tasks.Shell>
+      <command>OMERO_DIST=$HOME/OMERO.server
+if [ -e $OMERO_DIST ]; then
+
+	if [ -e $WORKSPACE/omeropy-virtualenv ]; then
+		source $WORKSPACE/omeropy-virtualenv/bin/activate
+        eval $(bash /opt/multi-config.sh ice3.5)
+    	$OMERO_DIST/bin/omero web stop
```

Comment:
Not yet. At the moment, the docker-compose creates a single web container which is re-used for multiple runs. We'll need to look into new jenkins plugins in order to enable the "spin up a fresh container each time" workflow. Definitely for it, but I think we should make the migration to omero-install atomically and then look at the bigger step to a many-container system.